packages/imperative/src/imperative/src/plugins/cmd/list/list.handler.ts

```diff
@@ -98,7 +98,7 @@ export default class ListHandler implements ICommandHandler {
 
         if(containsLegacyPlugin)
         {
-            listOutput = listOutput + `${chalk.yellow.bold("Plug-ins marked with (?) may be invalid or out of date, and might need to be reinstalled.")}` + "\n";
+            listOutput += chalk.yellow.bold("Plug-ins marked with (?) may be invalid or out of date, and might need to be reinstalled.") + "\n";
```

Comment:
I wonder what's more efficient (memory-wise) 😋
either way I think it's fine for this one case 😋